app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -800,14 +839,36 @@ class PipelineSampleReport extends React.Component {
         taxLevel={taxInfo.tax_level}
         taxName={taxInfo.name}
         ncbiEnabled={ncbiEnabled}
-        onNcbiActionClick={this.gotoNCBI}
+        onNcbiActionClick={withAnalytics(
+          this.gotoNCBI,
+          "PipelineSampleReport_ncbi-link_clicked",
+          analyticsContext
+        )}
         fastaEnabled={fastaEnabled}
-        onFastaActionClick={this.downloadFastaUrl}
+        onFastaActionClick={withAnalytics(
+          this.downloadFastaUrl,
+          "PipelineSampleReport_fasta-link_clicked",
```

Comment:
maybe `taxon-fasta-link` because many links could be called a fasta link on this page